translation.rst

```diff
@@ -307,10 +307,10 @@ parts of your application and mocking it in your tests.
 
 Instead of translating a string at the time of creation, you can use a
 "translatable object", which is an instance of the
-:class:`Symfony\\Component\\Translation\\TranslatableMessage` class. This object stores
+:class:`Symfony\\Component\\Translation\\Translatable` class. This object stores
 all the information needed to fully translate its contents when needed::
 
-    use Symfony\Component\Translation\TranslatableMessage;
+    use Symfony\Component\Translation\Translatable;
 
     // the first argument is required and it's the original message
     $message = new TranslatableMessage('Symfony is great!');
```

Comment:
we need to change it here and below too

Reply:
Good catch! Done in https://github.com/symfony/symfony-docs/commit/8d1364ecfb4db767168dcd33dcb045971ffc4ff6 Thanks!